src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/SessionDictionary.cs

```diff
@@ -0,0 +1,471 @@
+            _issuer = Add("Issuer");
+            _originalIssuer = Add("OriginalIssuer");
+            _issuerRef = Add("IssuerRef");
+            _claimCollection = Add("ClaimCollection");
+            _actor = Add("Actor");
+            _claimProperty = Add("ClaimProperty");
+            _claimProperties = Add("ClaimProperties");
+            _value = Add("Value");
+            _valueType = Add("ValueType");
+            _label = Add("Label");
+            _type = Add("Type");
+            _subjectId = Add("subjectID");
+            _claimPropertyName = Add("ClaimPropertyName");
+            _claimPropertyValue = Add("ClaimPropertyValue");
+            _authenticationType = Add("AuthenticationType");
+            _nameClaimType = Add("NameClaimType");
+            _roleClaimType = Add("RoleClaimType");
+            _nullValue = Add("Null");
+            _emptyString = Add(String.Empty);
+            _key = Add("Key");
+            _effectiveTime = Add("EffectiveTime");
+            _expiryTime = Add("ExpiryTime");
+            _keyGeneration = Add("KeyGeneration");
+            _keyEffectiveTime = Add("KeyEffectiveTime");
+            _keyExpiryTime = Add("KeyExpiryTime");
+            _sessionId = Add("SessionId");
+            _id = Add("Id");
+            _validFrom = Add("ValidFrom");
+            _validTo = Add("ValidTo");
+            _contextId = Add("ContextId");
+            _sesionToken = Add("SessionToken");
+            _sesionTokenCookie = Add("SessionTokenCookie");
+            _bootStrapToken = Add("BootStrapToken");
+            _context = Add("Context");
+            _claimsPrincipal = Add("ClaimsPrincipal");
+            _windowsPrincipal = Add("WindowsPrincipal");
+            _windowsIdentity = Add("WindowIdentity");
+            _identity = Add("Identity");
+            _identities = Add("Identities");
+            _windowsLogonName = Add("WindowsLogonName");
+            _persistentTrue = Add("PersistentTrue");
+            _sctAuthorizationPolicy = Add("SctAuthorizationPolicy");
+            _right = Add("Right");
+            _endpointId = Add("EndpointId");
+            _windowsSidClaim = Add("WindowsSidClaim");
+            _denyOnlySidClaim = Add("DenyOnlySidClaim");
+            _x500DistinguishedNameClaim = Add("X500DistinguishedNameClaim");
+            _x509ThumbprintClaim = Add("X509ThumbprintClaim");
+            _nameClaim = Add("NameClaim");
+            _dnsClaim = Add("DnsClaim");
+            _rsaClaim = Add("RsaClaim");
+            _mailAddressClaim = Add("MailAddressClaim");
+            _systemClaim = Add("SystemClaim");
+            _hashClaim = Add("HashClaim");
+            _spnClaim = Add("SpnClaim");
+            _upnClaim = Add("UpnClaim");
+            _urlClaim = Add("UrlClaim");
+            _sid = Add("Sid");
+            _referenceModeTrue = Add("ReferenceModeTrue");
+        }
+
+#pragma warning disable 1591
+        public static SessionDictionary Instance
+        {
```

Comment:
Make all these properties use expression style, then you don't need blank lines between each of them.